python/ml4ir/base/tests/test_monte_carlo_scorer.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from unittest.mock import MagicMock
+import tensorflow as tf
+import pathlib
+import yaml
+from ml4ir.base.model.scoring.monte_carlo_scorer import MonteCarloScorer
+
+
+class TestMonteCarloScorer(unittest.TestCase):
+
+    def setUp(self):
+        model_config_file = pathlib.Path(__file__).parent / "data" / "configs" / "model_config_monte_carlo_10.yaml"
+
+        with open(model_config_file.as_posix(), 'r') as f:
+            model_config = yaml.safe_load(f)
```

Comment:
@mohazahran you can get rid of this file and load and just pass a dictionary with the fields you need I guess.